wfe2/wfe.go

```diff
@@ -2111,7 +2111,7 @@ func (wfe *WebFrontEndImpl) NewOrder(
 		RegistrationID: acct.ID,
 		Names:          names,
 	})
-	if err != nil || order == nil || order.Id == 0 || order.CreatedNS == 0 || order.RegistrationID == 0 || order.ExpiresNS == 0 || len(order.Names) == 0 {
+	if err != nil || order == nil || order.Id == 0 || order.RegistrationID == 0 || len(order.Names) == 0 || order.Created == nil || order.Expires == nil { //core.IsAnyNilOrZero(order.Created, order.Expires) {
```

Comment:
Commented out code?

Reply:
Yup, that's a debugging mistake. Thank you.